windows/RNFS/pch.h

```diff
@@ -0,0 +1,3 @@
+﻿#pragma once
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Foundation.Collections.h>
```

Comment:
please add 
```cpp
#include <windows.h>
```

Since RNFS uses some jsi headers that rely on windows.h (see the issue that @creambyemute ran into)

Reply:
Just updated this.

Reply:
thanks a lot @avmoroz!